Module/STIG/Functions.Checklist.ps1

```diff
@@ -400,24 +421,19 @@ function Get-FindingDetails
 
     switch ($setting.ResourceID)
     {
-        {$PSItem -match "^\[(x)?Registry\]"}
-        {
+        {$PSItem -match "^\[(x)?Registry\]"} {
```

Comment:
Let's move this opening curly bracket so it's on it's own line and follows [style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#one-newline-before-braces).  Lets do the same for the rest of this switch statement.